app/Actions/FulfillTrade.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Actions;
+
+use App\Models\Entry;
+use App\Models\Trade;
+use App\Models\Item;
+use App\Models\Character;
+use App\Models\User;
+use App\Exceptions\TradeException;
+use Illuminate\Support\Facades\Auth;
+use Lorisleiva\Actions\Concerns\AsAction;
+
+class FulfillTrade
+{
+    use AsAction;
+
+    /**
+     * Expect an array of item details, create each item and attach them to the appropriate relation objects
+     *
+     * @param Entry $entry
+     * @param array $items
+     */
```

Comment:
oops your copy paste is showing... update this doc block please. also you can use `php artisan make:action ActionName` in the future instead

Reply:
oopsie

Reply:
fixed.